src/data/importJsonResume.ts

```diff
@@ -0,0 +1,117 @@
+import type { Writable } from 'svelte/store';
+import {
+  type BasicsStore,
+  workStore,
+  type Work,
+  basicsStore,
+  type Highlight,
+  educationStore,
+  type Education
+} from './data';
+import type { JsonResume } from './JsonResume';
+
+/**
+Imports the contents of a JSON Resume and adds the contents into the appropriate stores.
+
+@param jsonResume {JsonResume} The json resume as defined by the [following schema](https://jsonresume.org/schema/)
+@param [basics=basicsStore] {BasicsStore} The stores for the basic information
+@param [work=workStore] {Writable<Work[]>} The store for the work array
+@param [education=educationStore] {Writable<Education[]>} The store for the education array
+*/
+export default function importJsonResume(
+  jsonResume: JsonResume,
+  basics: BasicsStore = basicsStore,
+  work: Writable<Work[]> = workStore,
+  education: Writable<Education[]> = educationStore
+) {
+  importBasics(jsonResume, basics);
+  importWork(jsonResume, work);
+  importEducation(jsonResume, education);
```

Comment:
:warning: I forgot to import skills! :warning: 

I need to add support for skills in this file and in my tests.